src/node.cc

```diff
@@ -4074,10 +4074,13 @@ static void DebugEnd(const FunctionCallbackInfo<Value>& args) {
 
 inline void PlatformInit() {
 #ifdef __POSIX__
+#if HAVE_INSPECTOR
   sigset_t sigmask;
   sigemptyset(&sigmask);
   sigaddset(&sigmask, SIGUSR1);
   const int err = pthread_sigmask(SIG_SETMASK, &sigmask, nullptr);
+  CHECK_EQ(err, 0);
+#endif
```

Comment:
Can you write this as `#endif  // HAVE_INSPECTOR`?  (Note the two blanks before the comment.)

You can't really do the CHECK_EQ at this point because fds 0-2 might not be pointing to valid file descriptions yet.